homeassistant/components/livisi/climate.py

```diff
@@ -99,14 +99,15 @@ async def async_added_to_hass(self) -> None:
 
         await super().async_added_to_hass()
 
-        target_temperature = await self.coordinator.async_get_vrcc_target_temperature(
-            self._target_temperature_capability
+        target_temperature = await self.coordinator.async_get_device_state(
+            self._target_temperature_capability,
+            "setpointTemperature" if self.coordinator.is_avatar else "pointTemperature",
```

Comment:
Looks good, but I think climate/switch and refactoring should be preliminary PR.

Reply:
Done. Created https://github.com/home-assistant/core/pull/90227